src/app/mork/celery/celery_app.py

```diff
@@ -11,6 +15,21 @@
     ],
 )
 
+
+@signals.celeryd_init.connect
+def init_sentry(**_kwargs):
+    """Initialize Sentry SDK on Celery startup."""
+    if settings.SENTRY_DSN is not None:
+        sentry_sdk.init(
+            dsn=settings.SENTRY_DSN,
+            enable_tracing=True,
+            traces_sample_rate=settings.SENTRY_CELERY_TRACES_SAMPLE_RATE,
+            release=__version__,
+            environment=settings.SENTRY_EXECUTION_ENVIRONMENT,
+            max_breadcrumbs=50,
+        )
+
```

Comment:
What do you think to use `set_tag` to distinguish `celery` and `api` application on sentry ?

```
sentry_sdk.set_tag("application", "celery")
```

https://docs.sentry.io/platforms/python/enriching-events/tags/